src/timer/Timer.js

```diff
@@ -1,17 +1,19 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function () {
-  var instance = this;
-  var timerInterval = setInterval(function () {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    let instance = this.seconds;
+    const timerInterval = setInterval(() => {
+      if (instance === 0) {
+        clearInterval(timerInterval);
+      }
 
-    console.log(instance.seconds);
-    instance.seconds -= 1;
-  }, 1000);
-};
+      console.log(instance);
+      instance -= 1; // the linter does not like "instance--". WTF?
```

Comment:
The Airbnb lint rules probably disallowed this due to this weirdness in JavaScript with `--` and spacing: https://eslint.org/docs/rules/no-plusplus